src/phases/add-enemy-buff-modifier-phase.ts

```diff
@@ -0,0 +1,26 @@
+import BattleScene from "#app/battle-scene.js";
```

Comment:
Maybe it's not the right time but I think there's been a general effort to remove .js from the imports. 
```suggestion
import BattleScene from "#app/battle-scene";
```